_posts/2021-04-02-gs_00001_computer_setup.md

```diff
@@ -0,0 +1,439 @@
+higher the speed the better the performance as long it is supported by the CPU
+and motherboard. Additionally, there exists a maximal number of RAMs as well as
+a maximal amount of storage. For B450M PRO-VDH MAX, this is 4 RAMs with in total
+maximal 128 GB. By the way: 2 x 8 GB memories usually perform better than a
+single 16 GB memory since they can be accessed in parallel. If you want to save
+money you can start with a single 8 GB RAM and buy a second one later if you
+want to.
+
+I recommend combining SDD and HDD drives. The SDD is still more expensive so I
+use it only for the operating system and programs that I use frequently. 256 GB
+were enough for me so far. Data that I do not use that often like pictures of my
+last vacation, I store on an HDD. Here I bought 1 TB. If you find out that
+either the SDD or HDD is not enough you can buy a third drive later.
+
+Lastly, you need a power supply, a computer housing and some thermal paste. For
+the supply, you need to sum up the power requirements of your parts. Since there
+is not so much price difference in low power segments, you can just buy a 400 W
+power supply. This is enough even you add a low- to mid-range graphics card. As
+for housing, you can almost choose whatever you like. Keep an eye on the form
+factor and the provided interfaces. A housing with preinstalled fans makes it
+easier for you. The thermal paste needs to be put between the processer and the
+heat sink to guarantee optimal cooling. Sometimes the fan already has some
+thermal paste on it. Then, you do not need additional paste.
+
+So we have:
+
+| Part                                     | Price    |
+| ---------------------------------------- | -------- |
+| AMD Ryzen 3 3200G                        | 114€     |
+| B450M PRO-VDH MAX                        | 76€      |
+| Corsair Vengeance LPX 2x8GB DDR4 3000Mhz | 86€      |
+| Silicon Power SSD 256 GB                 | 33€      |
+| Seagate Barracuda HDD 1TB                | 36€      |
+| be quiet! 400W power supply              | 42€      |
+| Sharkoon VG5-V housing                   | 39€      |
+| Thermal paste                            | 5€       |
+| **Overall**                              | **431€** |
+
+Note: These are today's prices, which can change every day and differ between
+suppliers. Please do not understand it as an explicit recommendation. It is more
+an example to show you that it is not necessary to have a super high-end
+computer. Of course, you can spend more if you like, but you also can downgrade
+a bit and get even cheaper. For things we plan to do in this blog, a computer as
+shown in the table will be sufficient.
+
+# Windows, Linux, Mac?
+
+So far we talked only about the hardware and not the software running on it. The
+main software on a computer is the operating system. It has control of your
+system and manages all your programs such as your browser. Windows is probably
```

Comment:
"such as your browser."

Sounds weird. Maybe because it's only one example given?